File: Code Coach/Paint Costs/PaintCosts.java
import java.util.Scanner;

public class PaintCosts {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        double colors = input.nextDouble(), total = 40 + 5 * colors, tax = total / 10;
        System.out.println((int) Math.ceil(total + tax));
        input.close();
    }
}

File: Code Coach/Paint Costs/PaintCosts.cpp
#include <iostream>
#include <cmath>
using namespace std;

int main() {
    float colors, total, tax;
    cin >> colors;
    total = 40 + 5 * colors;
    tax = total / 10;
    cout << (int) ceil(total + tax);
    return 0;
}